src/reducers.js

```diff
@@ -73,14 +73,22 @@ function members(state = { byMid: {}, byUid: {} }, action) {
   }
 }
 
+/**
+ * Helper function to parse operations and add them to
+ * existing state.
+ */
+function parseOps(ops, opDocs) {
+  const newOps = opDocs.reduce((res, d) => {
```

Comment:
the variable names are kinda meaningless to me, why are these ops new, is this function actually parsing something?